_includes/join-us/join-us-contribute-to-strimzi-band.html

```diff
@@ -0,0 +1,120 @@
+
+      <p>Another way to spread the word about Strimzi is by writing blog posts. 
+        If you have some interesting topics. 
+        For example about how you use Strimzi and your Strimzi use cases or about using Strimzi with other projects and technologies. 
+        You can <a href="https://strimzi.io/blog/2021/07/29/how-to-write-blog-posts-for-strimzi-blog/">write a blog post</a> for our <a href="https://strimzi.io/blog/">own blog</a> which is on the Strimzi website. 
+        But you can also publish it on your personal or company blog and let us know and we can help you share it with the Strimzi community.</p>
+
+      <p>Talking at conferences or meetups is another great way to spread the word about Strimzi. 
+        And it doesn’t matter whether it is a small local meetup or a big conference for thousands of attendees. 
+        Let us know about your talks and we can share them with the community. 
+        If you want, we can also help to review your materials or conference proposals. 
+        If you are an organizer of a conference or meetup and want to have someone to talk about Strimzi, we can also try to help you find the right speaker.</p>
+
+      <h3>Ask for Help or Help Others</h3>
+
+      <p>If you are new to Strimzi, you might have many questions and or run into some beginner issues. 
+        We hope that our <a href="https://strimzi.io/quickstarts/">quickstarts</a>, our <a href="https://strimzi.io/documentation/">documentation</a> or the <a gref="https://www.youtube.com/playlist?list=PLpI4X8PMthYeCSpy9a-mGtvDbMgqGNYNy">tutorials</a> on our YouTube channels make it easy to get started. 
+        But don’t be afraid to ask if you have any questions or issues. 
+        The <a href="https://slack.cncf.io/"><code>#strimzi</code> Slack channel</a>, the <a href="https://lists.cncf.io/g/cncf-strimzi-users">user mailing list</a> or the <a href="https://github.com/strimzi/strimzi-kafka-operator/discussions">GitHub discussions</a> are the best places to ask questions or ask for help.</p>
+
+      <p>To get some useful and fast help, it is always good to cover the following points:</p>
+      
+      <ul>
+      <li>Check the documentation and discussion history before asking new question.</li>
+      <li>Explain what Strimzi and Kubernetes version you are using and in which environment you are running it. 
+        There might be known issues or incompatibilities.</li>
+      <li>Include as many logs and configurations as possible! 
+        A single log line or exception or just a small snippet of your custom resource might not be enough to understand the problem and to help. 
+        Ideally either share them directly in your question / issue formatted as code or as an attachment. 
+        Keep in mind that people might not be eager to help if they have to decipher the logs from an upside-down picture of your screen which you took with your smartphone. 
```

Comment:
I know this is probably coming from bitter experience, but this line might be a bit strong. :)

Reply:
I agree with @merlante but at same time I know from where it comes. We had a lot of "bad" experiences with users sharing logs this way. Anyway, yeah ... I would remove it or using a sentence where you mention that for having better help, providing all the possible logs in text format is a way to move further easily.